src/ScottPlot.Demo/PlotTypes/Bar.cs

```diff
@@ -27,11 +27,10 @@ public void Render(Plot plt)
                 }
 
                 // make the bar plot
-                plt.PlotBar(Xs, dataA);
-                plt.PlotErrorBars(Xs, dataA, null, null, errorA, null);
+                plt.PlotBar(Xs, dataA, errorY: errorA);
 
                 // customize the plot to make it look nicer
-                plt.Axis(-10, 110, 0, 110);
+                plt.Axis(null, null, 0, null);
```

Comment:
I changed these to help me debug. For WPF, the null defaults made the plots super zoomed in on the first bar. I thought I had screwed up the code until I zoomed out. Just a heads up.

Reply:
This might be a bug that just got introduced by #313... I'll take a closer look! Other cookbook have messed up auto-axis suddenly